
# GMCT - 2022 CONTEXT 
Tag: #Manuscript

Journal : Trends in ecology and evolution 
Type: opinion manuscript
https://www.cell.com/trends/ecology-evolution/authors
Schedule :  [GMTC To do list ](Goals_GMCT.md)
  
Author 
Highlights 
Abstract

 
**

Successful public health strategies for mitigating emerging infectious disease, requires understand evolutve history from the interaction between pathogens with their host under a geographical frame.  Using the coevolutionary hipothesis proposed by Thompson , will lead us to explain the evolutve mechanism behind virulence diversity ( refering to virulence the amount and the speed of which a pathogen kill a host ) and inmune adaptation in a biogeographical scale. Coevolution is the reciprocal adaptation among interacting organisms, and one of the teories that better explain biodiversity because coevolutionary selection can lead to speciation. Geographic mosaic theory of coevolution examines geographic variation in species interactions to approaching a method to measure coevolution.

However, 


A geographic mosaic of coevolution (GMC) occurs when local
variation in species interactions produces a spatially variable
pattern of reciprocal adaptation in these species


Measuring coevolution is difficult because it involves
determining how two species interact through time, and without a time machine it is difficult to measure how traits of
interacting species have changed through time. The difficulties
in rigorously testing coevolutionary hypotheses led to questions
about its importance as an evolutionary force. To solve the
problems in testing the coevolutionary hypothesis, Thompson
(1999, 2005) developed the geographic mosaic theory of coevolution that examines geographic variation in species interactions to use space as a proxy for time to measure coevolution.
A geographic mosaic of coevolution (GMC) occurs when local
variation in species interactions produces a spatially variable
pattern of reciprocal adaptation in these species

Measuring factors as local adaptation, tranfering some genes, biogeografical evidence. 
Acordingly, we propose  *methods* to measure pathogen - host coevolution, trying to encompass the complexity of coevolution. 

Local coevolutionary process between pathogen - host can regulate the pathogens virulence according to its  specialization to the inmune sytem of the host, despite how promising seems using GMTC with epidemiological porpuses. Its implementation is still challenging, considering that coevolution implies rules as properly measuring  coevolution, based on local and spatial criteria that frequently are both hard to find and measure. 

**Coevolution of pathogen- host interaction**





*Geographic mosaic for pathogen host coevolution

Acordingly, at a local level,  there are several factors driving pathogen host interaction : 
- [ ] Theories of Pathogen host coevolution - See the ones mentioned un one of your QE questions proposal
pathogens can respond in a variety of ways, and there have been several conceptual models (gene for gene, matching alleles, red queen, fluctuating selection dynamics, ...)

- [ ] Paragraph about  evolution virulence : VIrulence is defined as the capacity of a pathogen of decrease the fitness of the host, but for several models have been adresses as the death rate once it is infected 
[[Cressler2016AdaEvo]]  ;  [[Frank2008MecPat]]  . The understanding of virulence and how we measure it has been approached for other researchers, but most of their research is model focused. 

Infection cycles occur at a geografical scale, with factors not only pathogen-host related. 

Coevolution implies consider populations and factors surrounding them.  Coevolution is defined as reciprocal evolutionary change that occurs between interacting groups [REF]. And the geographic mosaic of coevolution [REF], measures coevolutionary selection, considering the presence or absence of certain traits in interacting species (trait remixing), the influence of external factors (selection mosaic), and how it can be evidenced at a geographical scale with the presence of hotspot and coldspot of coevolution. [REF thompson].  
- [ ] Insert bref resumen about thompson hypothesis  

**
**Why do we need to Identifyng hotspots of pathogen -host coevolution?**


Take home message: Hotspot areas we can find pathogens that can be highly virulent, and not necessary specialized.  Recognizing to avoid being part of it (Main idea: Si detectamos los hotspot, podremos saber como infuencian en otras areas y por lo tanto evitar o mitigar su impacto ) 

- [ ] First parragraph. We dont want to be part of a hotspot , we want to live in coldspots with generalist pathogens. Personal opinion

En los hotspot, tenemos interacciones altamente especializadas y si saltan a otra especie, podrian generar caos . Generalista o ser especialista 


- [ ] Evolution of virulence. Highly specialized and highly viruelnt or generalist with low virulence. 
Coevolutionary cold spots arise for many reasons, including nonoverlapping geographic ranges (Nuismer et al., 2003), the presence or absence of additional species (Benkman et al., 2001) or shifting abiotic conditions (Hochberg and van Baalen, 1998).
1. Interacciones P-H especializadas. Extincion del pathogeno, maladaptacion o cross species trasnmission 
- [ ] Recognize transmission cyle and how to be part of it. (So we can avoid them)
(Cuando hablamos de coevolucion pathogeno - huesped. Como se ve eso en la naturaleza de acuerdo a Thompson) 1. Como se ve un hotspot : Un hotspot, de acuerdo a thompson, teoricamente lo podriamos identificar si - Insertar lo que dice G sobre identficar estas areas- 

3. Cross species transmission. 
Queremos erradicar los patogenos que tienen un impacto en la calidad de vidad de nuestro huesped de interes, en este caso los seres humanos. Tambien tenemos los patogenos que no nos interesa erradicar por que son imperceptibles en su impacto a la salud, o a la economica global.  Hemos logrado erradicar a microorganismos especializados en un solo hospedero. La importancia de identificar los patogenos que sean especializados pero facilmente se muevan a otra especie o aquellos que sean generalistas pero que a su vez generen decrimento del bienestar del individuo que infecta

- [ ] It's not only about hotspot, it's also about how it interact with cold spot 
3. Como influencia los hotspot en los coldspot ,  Si bien es cierto pueden existir maladaptaciones, por lo tanto, estos patogenos no logran infectar nada. Tambien puede ocurrir que estos patogenos sean muy virulentos (despues de estar en el red queen theory con otros huespedes) por lo tanto, una vez pasen a un nuevo huesped, sea letal.  [[Gomulkiewicz2007DosDon]]  [@Gomulkiewicz2007DosDon] [[Gomulkiewicz2007DosDon]] 
**

**How we identifyng coevolutive hotspots?**

Actualmente, tenemos metodos que nos permited aproximarnos mucho mas a la identificacion de estos metodos. 
Por ejemplo: 
 - [ ] Aqui incluimos algo de los metodos de betacov 

Mientras que @Gomulkiewicz2007DosDon decia esto , tambien @Woolhouse2002BioBio dice esto. 

- [ ] Resumen de la idea de [[Woolhouse2002Biomedicalcoevolution ]] y [[Gomulkiewicz2007DosDon]] 
**
- [ ] 
@Woolhouse2002BioBio

The identification of coevolutive pathogen-host areas requires evidence of change… - *Awesome methods?*
But we still face some challenges like 
However, gene flow can also be a creative force, providing genetic variation for reciprocal selection and continued coadaptation
under antagonistic coevolution, theoretical studies show that, provided migration rates are low, the species with the relatively greater amount of gene flow will generally show
adaptation, rather than maladaptation [@Gomulkiewicz2007DosDon] (Gandon et al., 1996; Gandon, 2002). A number of empirical studies of hosts and parasites have examined local adaptation and relative gene flow and support this theoretical result (reviewed in Dybdahl and Storfer, 2003).
Whether gene flow is an important source of trait remixing depends on the relative geographic scales and strength of selection mosaics compared to gene flow
Pausa aqui - pagina 255 - discusion sobre el paso de genes a traves de diferentes patches. Por lo tanto explicar con algo de detalle eso 

2. Como cuantificamos que efectivamente es un hotspot 
Si queremos ,quantify levels of local maladaptation in interacting species , accordingly, un sistema inmune que no este adaptado lo suficiente seria mal adaptado?
Neither trait mismatching nor local maladaptation provide a rigorous test of the GMTC
 it is important to emphasize that – at least from a theoretical perspective – selection mosaics and coevolutionary cold spots can greatly increase the likelihood of observing local maladaptation or trait mismatching

Este autor propone que : To sum up, our dos and don’ts for evaluating hot and cold spots are: do use regression approaches, selective source analysis and multivariate approaches to infer hot spots. When looking to establish cold spots for a particular interaction, don’t forget to include regions where one of the species is completely absent.

*Ok, How can we identify it:* Probably, the most direct way to detect a selection mosaic empirically is to carefully replicate genotype frequencies in all species across different localities. @Gomulkiewicz2007DosDon


**ALSO - Concluding remarks ** 

Here we have theoretical work may identify additional predicted patterns that prove exclusive to the GMTC.
We also should measure how the fitness of at least one specie is afected 

- [ ] We want to know where are the risky areas of hotspot, to see which pathogens can rise from there and be prrepared for cross species transmission 
**
